bentoml/service/inference_api.py

```diff
@@ -189,6 +210,7 @@ def handle_error(tasks, status_code: int, error_message: str, exception: Excepti
                             err_msg=error_message,
                             err_class=exception_class_name,
                         )
+                    log_request()
```

Comment:
Instead of putting it twice in the code, wouldn't it be better to put it once at line 192?
Seems like it doesn't matter if it is a batch or not